florist/api/server.py

```diff
@@ -0,0 +1,118 @@
+"""FLorist server FastAPI endpoints."""
```

Comment:
(Again a bit of a nit so feel free to ignore if you think its overkill or disagree) I am wondering if it makes more sense to have the start_training endpoint in a file separate from the server since it really is not specific to the server, rather an endpoint about orchestrating the launch of a server and the clients.

Reply:
I think that makes sense. FastAPI has this concept about routes, which will be used for other services I am building, I can put it in a separate `training` route.